dali_mqtt_daemon.py

```diff
@@ -375,6 +376,56 @@ def on_message_brightness_cmd(mqtt_client, data_object, msg):
     except KeyError:
         logger.error("Lamp %s doesn't exists", light)
 
+def on_message_brightness_get_cmd(mqtt_client, data_object, msg):
+    """Callback on MQTT brightness get command message."""
+    logger.debug("Brightness Get Command on %s: %s", msg.topic, msg.payload)
+    light = re.search(
+        MQTT_BRIGHTNESS_GET_COMMAND_TOPIC.format(data_object["base_topic"], "(.+?)"),
+        msg.topic,
+    ).group(1)
+    try:
+        if 'group_' in light:
```

Comment:
This block is the same as `on_message_brightness_cmd`

could you please refactor, so we don't repeat this code ?